README.md

```diff
@@ -11,37 +11,49 @@ This is a library which allows XMonad to manage Mac OS X windows.
 
 ## Installation
 
+**Note**: users with macOS 10.8 (Mountain Lion) and superior will have to
```

Comment:
Why is XQuartz needed?

Reply:
I have not tried to compile the code without XQuartz (already installed on my machine), but I guess it would be hard to run it without (since X11 support has been abandoned since OS X Lion Moutain):
- https://en.wikipedia.org/wiki/OS_X_Mountain_Lion#Dropped_and_changed_features
- https://support.apple.com/en-us/HT201341

I can try to test this on a fresh installation!